python/mxnet/test_utils.py

```diff
@@ -1411,8 +1411,29 @@ def read_data(label_url, image_url):
             'test_data':test_img, 'test_label':test_lbl}
 
 def get_bz2_data(data_dir, data_name, url, data_origin_name):
-    """Download and extract bz2 data."""
+    """Download and extract bz2 data.
+
+    Parameters
+    ----------
+
+    data_dir : str
+        Absolute or relative path of the directory name to store bz2 files
+    data_name : str
+        Name of the output file in which bz2 contents will be extracted
+    url : str
+        URL to download data from
+    data_origin_name : str
+        Name of the downloaded b2 file
+
+    Examples
+    --------
+    >>> get_bz2_data("data_dir", "kdda.t",
+                     "https://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/binary/kdda.t.bz2",
+                     "kdda.t.bz2")
+    """
+
     download(url, dirname=data_dir, overwrite=False)
+    cwd = os.path.abspath(os.getcwd())
     os.chdir(data_dir)
```

Comment:
Can we dump chdir and modify the paths instead?

Reply:
Removed chdir and used paths.